.babelrc

```diff
@@ -1 +1 @@
-{"presets": ["module:metro-react-native-babel-preset"]}
+{"presets": ["module:metro-react-native-babel-preset", "@babel/flow"]}
```

Comment:
I'll revert this.  I was trying to set up flow so that it would work without having to use comments, but `ncc` wasn't liking that.